src/FullBuild/Program.fs

```diff
@@ -49,12 +49,12 @@ let tryMain argv =
     // view
     | Command.AddView viewInfo -> Commands.View.Add viewInfo
     | Command.DropView viewInfo -> Commands.View.Drop viewInfo.Name
-    | Command.ListViews -> Commands.View.List ()

```

Comment:
Have never used it. The first reflex is to recreate the view even if it already exists :)